_posts/2020-10-13-community-call-dei.md

```diff
@@ -0,0 +1,22 @@
+---
+
+On September 10, 2020, the US-RSE Organization held a community call on "US-RSE's role in promoting Diversity, Equity and Inclusion". The goal of the call was to have an open discussion about what steps US-RSE might take in promoting diversity in our profession, both within the US-RSE organization and externally through outreach to the broader community, and if the participants in the call supported the idea of taking such steps.
+
+We initially talked about the fact that [US-RSE's Code of Conduct](https://us-rse.org/code-of-conduct/) includes some discussion of diversity, but that our mission does not currently include diversity, so in essence, we say that we are open to and supportive of diversity but we are not actively encouraging it either in our organization or in the RSE profession.
+
+It was suggested that members take a look at the [slides](https://doi.org/10.6084/m9.figshare.12955094.v4) from a talk given by Neil Chue Hong, Director of the Sustainability Software Institute. Neil talked about the lack of diversity within research software engineering, even when compared to academia as a whole or computer science. This led to a rich discussion about possible future directions and initiatives to improve diversity and inclusion:
+
+-   Adding a page to our website with resources for members who want to learn about diversity and inclusion in general, as well as practices they can use in their own lives. An example of such resources is guidelines on how to write good references for students while avoiding gender bias.
+
+-   Making sure we are all inclusive in the hiring process without giving applicants the perception that they are only invited because they belong to a minority, or perpetrating a "diversity hire". Broadening the pool of interested applicants through education about the field might prevent this, which highlights the importance of outreach outside our immediate community.
+
+-   Outreach about the RSE profession to on-campus groups with student participation. It was noted that since events are virtual these days, that now might be a good time to try this form of outreach.
+
+We also discussed the RSE Stories podcast as a current success for community building and increasing the representation of diverse voices, and the potential to expand into topics of diversity and inclusion. It was brought up that groups use social media platforms differently, and that outreach should include written and spoken communication with disability access in mind. A new channel on the US-RSE slack, [\#dei-discussion](https://usrse.slack.com/archives/C01C8CJQ7AP), was created to continue further discussion. This channel could be a place for also sharing individual activities that others could get involved in. Numerous participants on the call mentioned activities that they were personally involved in that were of interest to other members.
+
+![Community Call about Diversity and Inclusion](/assets/posts/2020-10-13-community-call-dei.png)
```

Comment:
```suggestion
![Community Call about Diversity and Inclusion]({{ site.baseurl }}/assets/posts/2020-10-13-community-call-dei.png)
```

Reply:
Actually, I think it should be
![Community Call about Diversity and Inclusion]({{ site.baseurl }}/assets/posts/2020-10-13-community-call-dei.png "Community Call about Diversity and Inclusion")